pages/buildkite_compute/overview.md

```diff
@@ -0,0 +1,82 @@
+# What is compute
+
+Buildkite compute provides a fully managed platform for you to run your agents.
+
+With Compute Services, the infrastructure management tasks traditionally handled by your team, such as provisioning, scaling, and maintaining the servers that run your agents, can now be managed by Buildkite.
+
+Buildkite compute is currently in private trials, you need to contact support to express interest and have the service switched on for your organization.
+
+## Compute types
+
+During our private trial phase, we are offering both Mac and Linux agents. We plan to extend our services to include Windows agents by late 2024 as part of our ongoing commitment to providing a comprehensive range of options.
+
+For detailed information about available agent sizes and configuration please see [Mac Compute Instances](/docs/buildkite-compute/macos-instances), and [Linux Compute Instances](/docs/buildkite-compute/linux-instances)
+
+Usage of all instance types is billed on a per-minute basis.
+
+Every Buildkite hosted agent within a cluster benefits from hypervisor-level isolation, ensuring robust separation between each instance.
+
+## Creating a compute queue
+
+You can set up distinct compute queues, each configured with specific types and sizes to efficiently manage jobs with varying requirements.
+
+For example you may have two queues setup
+
+* `mac_small_7gb`
+* `mac_large_32gb`
+
+Learn more about best practice queue configuration [here](/docs/clusters/overview#clusters-and-queues-best-practice-how-should-i-structure-my-queues)
```

Comment:
As a general rule, probably best to avoid creating links against non-descriptive words like 'here' and instead, generate against the core descriptive text/words itself.

```suggestion
Learn more about [best practices for configuring queues](/docs/clusters/overview#clusters-and-queues-best-practice-how-should-i-structure-my-queues).
```